user-manuals/facilities/manage-staff-and-locations.md

```diff
@@ -1,25 +1,24 @@
 # Manage Locations
 
-Effective stock management relies on proficient location management.\
-To enable store associates to establish precise locations for managing stock which will ease in receiving and storing inventory, as well as facilitating the picking process, ​​retailers can define specific locations within facilities, such as Aisle, Section, Level, or Position. Users can manage locations by the following steps:
+Effective stock management relies on proficient location management. To enable store associates to establish precise locations for managing stock, which will ease in receiving and storing inventory, as well as facilitating the picking process, ​​retailers can define specific locations within facilities, such as Aisle, Section, Level, or Position. Users can manage locations in the facility using the `Locations` tab in the footer on the `Facility details` page. 
 
-**Add Locations**
+&nbsp;
```

Comment:
what are these special lines? @sanjanashrivastava

Reply:
Initially it looked a little closely spaced to me, so I was checking how it would look with line breaks. Edited it, please check now.